### PR TITLE
FC-1102 Adds fuels cost for some :select operations in analytical queries

### PR DESCRIPTION
**Pull request by bplatz**
Analytical queries did not add any fuel cost for their operations performed in the :select clause.

This adds fuel costs for:
selectDistinct
orderBy
expandMaps

Still know we don't have fuel costs for:
aggregate function processing
groupBy - which uses a different process for results.

### CODE REVIEW

**Review by ldw1007 — APPROVED** [review]

File: src/fluree/db/query/fql.cljc [hunk 1]
```diff
@@ -720,17 +715,26 @@
...
 
 (defn order-result-tuples
-  [headers orderBy tuples]
+  "Sorts result tuples when orderBy is specified.
+
+  Operation should happen before tuples get filtered, as the orderBy variable might
+  not be present in the :select clause.
+
+  Order operation = 2 fuel per tuple ordered."
+  ;; TODO - check/throw max fuel
```
Comment (cap10morgan) [inline src/fluree/db/query/fql.cljc:724]: Did you intend to do this now or later?

**Review by cap10morgan — APPROVED** [review]
One question about a TODO but looks good to me overall!

File: src/fluree/db/query/fql.cljc (hunk 1 above)
Comment (bplatz) [reply src/fluree/db/query/fql.cljc:724]: Later...

**Review by cap10morgan — APPROVED** [review]
NOCASE addition looks good to me!

**Pull request merged**